src/pkg/cli/client/byoc/baseclient.go

```diff
@@ -122,7 +131,20 @@ func (b *ByocBaseClient) GetProjectDomain(projectName, zone string) string {
 	return domain
 }
 
+const TIER_ERROR_MESSAGE = "current subscription tier does not allow this action: "
+
+type ErrNoPermission string
+
+func (e ErrNoPermission) Error() string {
+	return TIER_ERROR_MESSAGE + string(e)
```

Comment:
Any reason not to use `fmt.Errorf` here?
```suggestion
	return fmt.Errorf("current subscription tier does not allow this action: %w", e)
```